### DISCUSSION TITLE
Any way to disable server statistics/debug info on /?

### DISCUSSION THREAD

**Discussion by untitaker** [DIYgod/RSSHub#12930] [Q&A] [answered]
### What feature is it?

the debug section on / exposes a lot of information about what profiles/users the rsshub instance is used for folllowing, and for anything but the demo rsshub.app instance, this can be sensitive data. is there a simple ay to block this data from appearing without just blocking access to / in the load balancer?

### What problem does this feature solve?

/ shows lots of information about specific URLs added that can potentially expose their operator or their social circle

### This is not a duplicated feature request or new RSS proposal

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this feature has not already been requested and this is not a [new RSS proposal](https://github.com/DIYgod/RSSHub/issues/new?assignees=&labels=RSS+proposal&template=rss_request_en.yml).

**Comment by untitaker** [comment]
side-note: searching for duplicated issues is hard for me because most issues are in a language i don't speak.

**Comment by TonyRL** [comment, marked as answer]
https://docs.rsshub.app/en/install/#configuration-logging-configurations
